src/GestureHandlerRootView.android.tsx

```diff
@@ -3,6 +3,7 @@ import * as React from 'react';
 import { PropsWithChildren } from 'react';
 import { requireNativeComponent, ViewProps } from 'react-native';
 import { isFabric } from './utils';
+import { tryInitializeFabric } from './init';
 
 const GestureHandlerRootViewNativeComponent = isFabric()
```

Comment:
Instead of calling `isFabric()` in top-level, let's check React Native version using `import { version } from 'react-native/package.json';`.